docker-compose.yml

```diff
@@ -4,7 +4,10 @@ services:
   postgres:
     image: postgres
     restart: always
-    env_file: .postgres.env
+    environment:
```

Comment:
In my mind the env_file is way better, but accepted.

Reply:
Please delete .api.env completely

Reply:
NO!